client/my-sites/post-type-filter/index.jsx

```diff
@@ -75,70 +80,100 @@ const PostTypeFilter = React.createClass( {
 			}
 
 			return memo.concat( {
-				count: jetpack ? null : count,
 				key: `filter-${ status }`,
-				path: [
-					'/types',
-					query.type,
+				// Hide count in all sites mode; and in Jetpack mode for non-posts
+				count: ( ! siteId || ( jetpack && query.type !== 'post' ) ) ? null : count,
+				path: compact( [
+					query.type === 'post'
+						? '/posts'
+						: '/types/' + query.type,
+					query.type === 'post' && query.author && 'my',
 					pathStatus,
 					siteSlug
-				].filter( Boolean ).join( '/' ),
+				] ).join( '/' ),
 				selected: mapPostStatus( pathStatus ) === query.status,
 				children: label
 			} );
 		}, [] );
 	},
 
 	render() {
-		const { siteId, query, jetpack } = this.props;
+		const { authorToggleHidden, jetpack, query, siteId, statusSlug } = this.props;
 		const navItems = this.getNavItems();
 		const selectedItem = find( navItems, 'selected' ) || {};
 
+		const scopes = {
+			me: this.translate( 'Me', { context: 'Filter label for posts list' } ),
```

Comment:
Wanna run the i18n codemod here? 😉  It'll actually please the linter, too.

Reply:
I've considered it -- that, and the `createClass` one. I ultimately decided to do that in a separate step since this PR is big enough already 😄